### PR TITLE
[5544] Numeric input "flips out" if max is lower than min

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5544

### CODE REVIEW

File: ui/app/src/env/i18n-legacy.ts [hunk 1]
```diff
@@ -427,6 +427,14 @@ export const contentTypesMessages = defineMessages({
   dependsOn: {
     id: 'contentType.dependsOn',
     defaultMessage: 'This property depends on "{dependency}"'
+  },
+  minValueError: {
+    id: 'contentType.minSizeError',
+    defaultMessage: "Minimum value can't be higher than maximum value."
```
Comment (rart) [inline ui/app/src/env/i18n-legacy.ts]:
```suggestion
    defaultMessage: "Minimum value can't be higher than maximum value"
```

File: ui/app/src/env/i18n-legacy.ts [hunk 2]
```diff
@@ -427,6 +427,14 @@ export const contentTypesMessages = defineMessages({
...
+  },
+  maxValueError: {
+    id: 'contentType.maxSizeError',
+    defaultMessage: "Maximum value can't be lower than minimum value."
```
Comment (rart) [inline ui/app/src/env/i18n-legacy.ts]:
```suggestion
    defaultMessage: "Maximum value can't be lower than minimum value"
```

File: static-assets/components/cstudio-forms/controls/numeric-input.js [hunk 3]
```diff
@@ -198,6 +198,14 @@
           inputEl.size = prop.value;
         }
 
+        if (prop.name == 'Value') {
+          const value = JSON.parse(prop.value);
+          inputEl.maxValue = value.maxValue;
+          inputEl.setAttribute('max', value.maxValue);
+          inputEl.minValue = value.minValue;
+          inputEl.setAttribute('min', value.minValue);
+        }
```
Comment (rart) [inline static-assets/components/cstudio-forms/controls/numeric-input.js]: What's this for?

File: static-assets/components/cstudio-admin/mods/content-type-propsheet/minMax.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      valueEl.value = value;
+      valueEl.fieldName = this.fieldName;
+      containerEl.appendChild(valueEl);
+
+      $(valueEl).on('focus', function () {
+        valueEl.setAttribute('type', 'number');
+      });
+
+      $(valueEl).on('blur', function (e) {
+        valueEl.setAttribute('type', 'text');
```
Comment (rart) [inline static-assets/components/cstudio-admin/mods/content-type-propsheet/minMax.js:47]: What's the type switching for?

File: static-assets/components/cstudio-forms/controls/numeric-input.js (hunk 3 above)
Comment (jvega190) [reply static-assets/components/cstudio-forms/controls/numeric-input.js]: Not needed, removing

File: static-assets/components/cstudio-admin/mods/content-type-propsheet/minMax.js (hunk 4 above)
Comment (jvega190) [reply static-assets/components/cstudio-admin/mods/content-type-propsheet/minMax.js:47]: It's a behavior we have in our current sub-controls for the content-types. Basically to show the numeric controls (increase-decrease arrows) only when the input is focused.

**Review by rart — APPROVED** [review]

**Pull request merged**